app/components/sidebars/main/channels_list/list/list.js

```diff
@@ -255,21 +339,12 @@ export default class List extends PureComponent {
     };
 
     renderItem = ({item}) => {
-        return (
-            <ChannelItem
-                channelId={item}
-                isFavorite={this.props.favoriteChannelIds.includes(item)}
-                navigator={this.props.navigator}
-                onSelectChannel={this.onSelectChannel}
-            />
-        );
-    };
+        const {unreadChannelIds} = this.props;
 
-    renderUnreadItem = ({item}) => {
         return (
             <ChannelItem
                 channelId={item}
```

Comment:
Looks like we're missing the `isFavorite` property here? It's still used by the connector for the `ChannelItem` in computing `shouldHideChannel`.